tools/dm/deployment.md

```diff
@@ -0,0 +1,463 @@
+---
+title: DM-Ansible 数据迁移部署方案
+summary: 使用 DM-Ansible 部署 DM 集群
+category: tools
+---
+# 使用 DM-Ansible 部署 DM
+
+DM-Ansible 是 PingCAP 基于[Ansible](https://docs.ansible.com/ansible/latest/index.html)的 [Playbooks](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html#about-playbooks) 研发的集群部署工具。本文将展示如何使用DM-Ansible 快速部署 Data Migration (DM)。
+
+## 准备工作
+
+在开始之前，先确保您准备好了以下配置的机器：
+
+1. 若干迁移目标机器，配置如下：
+
+    - CentOS 7.3 (64-bit) 或更高版本, x86_64 架构 (AMD64)
+    - 联机网络
+    - 关闭防火墙，或者设置服务端口
+
+2. 一台控制机器，配置如下：
+
+    - CentOS 7.3 (64-bit) 或更高版本， 安装 Python 2.7
+    - Ansible 2.5 或后续版本
+    - 互联网接入
+
+## Step 1: 在控制机器上安装依赖关系
+
+> **注意：**请确保使用 `root` 账户登陆控制机器。
+
+根据控制机器的操作系统版本，运行相应命令。
+
+- CentOS 7：
+
+    ```
+    # yum -y install epel-release git curl sshpass
+    # yum -y install python-pip
+    ```
+
+- Ubuntu:
+
+    ```
+    # apt-get -y install git curl sshpass python-pip
+    ```
+
+## Step 2: 在控制机器上创建 `tidb` 用户，并生成 SSH 密钥
+
+> **注意：**请确保您此时使用 `root` 账户登陆控制机器。
+
+1. 创建 `tidb` 用户。
+
+    ```
+    # useradd -m -d /home/tidb tidb
+    ```
+
+2. 为  `tidb` 用户设置密码。
+
+    ```
+    # passwd tidb
+    ```
+
+3. 在 sudo 文件尾部加上`tidb ALL=(ALL) NOPASSWD: ALL`，为 `tidb` 用户设置免密使用 sudo。
```

Comment:
Did you add some special chars in this line? You can view some � in the rendered markdown file.
more � below, and you can view the rendered markdown file to config the format of the doc.